app/views/feedback/new.html.haml

```diff
@@ -0,0 +1,15 @@
+
+%section#content.content-main.form_wrapper{role: "main"}
+  %h1 Your Feedback
```

Comment:
Well I copied it straight from the copy provided by the content design folk but sure I'll do that and let them know.